src/plopp/backends/matplotlib/line.py

```diff
@@ -159,8 +159,11 @@ def _make_line(
                 fmt="none",
             )
 
-        if self.label:
-            self._ax.legend()
+        if self.label and self._canvas._legend:
```

Comment:
For navigating users, how about throwing a warning if `label` is not set but `_legend` is `True`...?
i.e. With the instruction how to set `name` attribute to the data array ...?